src/js/actions/transform.js

```diff
@@ -390,9 +433,9 @@ define(function (require, exports) {
                 positions: []
             },
             layerOneActions = _getMoveLayerActions
-                .call(this, document, layers.get(0), newPositions.get(0), payload.positions),
+                .call(this, document, layers.get(0), newPositions.get(0), undefined, payload.positions),
```

Comment:
maybe you have to set the `referencePoint` for `_getMoveLayerActions` here; I tried the swap position and it raise an error.